osbs/tekton.py

```diff
@@ -556,6 +559,16 @@ def has_not_finished(self):
     def was_cancelled(self):
         return self.status_reason == 'PipelineRunCancelled'
 
+    def wait_for_finish(self):
```

Comment:
I would appreciate a docstring that explains how this method should be used - only for "almost finished" pipeline runs, not newly started ones.

Reply:
added